<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetUp</class>
 <widget class="QDialog" name="SetUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SetUp</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="text">
     <property name="font">
      <font>
       <family>AcadEref</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>修改监控软件</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="soft_1">
       <property name="text">
        <string>爱奇艺</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="soft_2">
       <property name="text">
        <string>QQ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="soft_3">
       <property name="text">
        <string>VS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="soft_4">
       <property name="text">
        <string>Android Studio</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="soft_5">
       <property name="text">
        <string>Wamp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="soft_6">
       <property name="text">
        <string>sublime_text</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="soft_7">
       <property name="text">
        <string>Dev-C++</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="soft_8">
       <property name="text">
        <string>eclipse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="soft_9">
       <property name="text">
        <string>IntelliJ IDEA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="soft_10">
       <property name="text">
        <string>PyCharm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="soft_11">
       <property name="text">
        <string>myeclipse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="soft_12">
       <property name="text">
        <string>微信web开发者工具</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="soft_13">
       <property name="text">
        <string>Adobe Photoshop CS6</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="ok_btn">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_btn">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
